frontend/packages/console-app/src/__tests__/network-policies/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - jotak
+  - mariomac
+  - OlivierCazade
+approvers:
+  - jotak
+labels:
+  - component/network-policies
```

Comment:
You'll need to add the label to openshift/release so it's available in the repo.

https://github.com/openshift/release/blob/master/core-services/prow/02_config/_labels.yaml#L548-L549

Reply:
Oh, then I've removed it to not block this PR. I'm not sure if this is really something we need.